core/src/main/java/com/yahoo/imapnio/async/request/ImapClientConstants.java

```diff
@@ -12,6 +11,9 @@
     /** String for CR and LF. */
     public static final String CRLF = "\r\n";
 
+    /** Byte array for CR and LF. */
+    public static final byte[] CRLF_B = { '\r', '\n' };
```

Comment:
fyi, arrays are bad for exposing as public variables. You should probably store as ByteBuf objects.

Reply:
Done.  Good point!

Reply:
After discussing regarding the constraints of ByteBuf or ByteBuffer,  I have moved this array private to individual classes.